sickbeard/search_queue.py

```diff
@@ -99,6 +99,12 @@ def is_dailysearch_in_progress(self):
                 return True
         return False
 
+    def is_forced_search_in_progress(self):
```

Comment:
Don't know if we need this? Can we perform a force search, while doing manual search/snatch?